xbmc/settings/AdvancedSettings.cpp

```diff
@@ -743,7 +743,7 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
     TiXmlElement* pVideoLatency = pElement->FirstChildElement("latency");
     if (pVideoLatency)
     {
-      float refresh, refreshmin, refreshmax, delay;
+      float refresh, refreshmin, refreshmax, delay, hdrlatency;
```

Comment:
Is the extra variable here really needed? It should be possible to pass `videolatency.hdrlatency` directly into `XMLUtils::GetFloat`.

Reply:
Is not the same because `XMLUtils::GetFloat` only retuns true when value in xml is correct in range [min, max]
then `videolatency.hdrlatency` only is updated when value is "valid".

Reply:
As far as I understand it returns `true` iff the value is found in the XML then updating the value and then clipping it.

Reply:
You're right

Reply:
Thank you. Done for both new and existing variables.